source/agora/consensus/protocol/Nominator.d

```diff
@@ -490,6 +496,7 @@ extern(D):
             this.log.trace(
                 "checkNominate(): Last block ({}) doesn't have majority signatures, signed={}",
                 this.ledger.height(), this.ledger.lastBlock().header.validators);
+            this.network.getMissingBlockSigs(this.ledger, this.acceptHeader);
```

Comment:
Doesn't this also have the yield problem ?

Reply:
But this one is only called when we are blocked because we do not have the signatures for the previous block. So it really is a priority to catch up before anything else.

Reply:
Alright